CHANGELOG.md

```diff
@@ -6,6 +6,127 @@ commits. For a comprehensive list, use the following command:
 git log --first-parent
 ```
 
+
+## Version 1.5.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.5.0. This release brings many important new features such as:
+- MPI-based multi-node support for all matrix formats and most solvers;
+- full DPC++/SYCL support,
+- functionality and interface for GPU-resident sparse direct solvers,
+- an inerface for wrapping solvers with automatic scaling and reordering applied,
```

Comment:
```suggestion
- an interface for wrapping solvers with scaling and reordering applied,
```

Reply:
I'm not sure automatic is the right adjective to describe this behavior, since it needs to be called explicitly.